airflow/www/views.py

```diff
@@ -2192,12 +2192,11 @@ class XComModelView(AirflowModelView):
 
     datamodel = AirflowModelView.CustomSQLAInterface(XCom)
 
-    base_permissions = ['can_add', 'can_list', 'can_edit', 'can_delete']
+    base_permissions = ['can_add', 'can_list', 'can_delete']
```

Comment:
I think we should remove "can_add" as well? When you restart the task, the xcom is wiped and you will be able to add it then ...

Reply:
Yeah, add has the same potential problems as edit

Reply:
Good point @potiuk

Reply:
Updated in https://github.com/apache/airflow/pull/9614/commits/550155c7fb793432521dc279ceca56ea0bd6e30e